source/common/http/header_map_impl.h

```diff
@@ -148,6 +133,8 @@ class HeaderMapImpl : public HeaderMap, NonCopyable {
   struct StaticLookupTable; // Defined in header_map_impl.cc.
 
   struct AllInlineHeaders {
+    void clear() { memset(this, 0, sizeof(*this)); }
```

Comment:
Can you also change the memset call in `HeaderMapImpl::HeaderMapImpl()` to call `inline_headers_.clear()` rather than memsetting directly? Thanks. I couldn't seem to comment on that line.